app/templates/index.html

```diff
@@ -92,33 +92,23 @@ <h2 class="marketplace-homepage-subheading">Find technology or people for digita
   </div>
     <div class="supplier-messages column-one-third">
       <aside role="complementary" aria-labelledby="supplier-message-heading">
-        
+
       {% if temporary_message or current_user.role == 'supplier' or dos_is_live %}
           <h2 id="supplier-message-heading">Sell services</h2>
       {% endif %}
-        
-      {% if temporary_message %}
-        {%
-          with
-            heading = temporary_message.heading,
-            subheading = temporary_message.subheading,
-            messages = temporary_message.messages,
-            message = temporary_message.message
-        %}
-          {% include "toolkit/{}.html".format(temporary_message.template_name) %}
-        {% endwith %}
-      {% endif %}
-        
+
       {% if dos_is_live %}
+
+      {% set opportunities %}
```

Comment:
It was to make things clearer, instead of repeating ifs.  They had to be broken up due to the way the messages are laid out in the template but it didn't read well.